Project/src/test/java/app/WebSecurityTest.java

```diff
@@ -34,13 +35,48 @@ public void loginProtection() throws Exception {
     }
 
     /**
-     * Tests path protection with different roles
+     * Tests open endpoints that dont require authentication
      * @throws Exception
      */
     @Test
-    public void protectedPath() throws Exception {
-        this.mockMvc.perform(get("/upload").with(user("test").password("pass").roles("SUBMITTER"))).andExpect(status().isOk());
-        this.mockMvc.perform(get("/upload").with(user("test").password("pass").roles("REVIEWER"))).andExpect(status().isForbidden());
+    public void openEndpoints() throws Exception {
+        this.mockMvc.perform(get("/register")).andExpect(status().isOk());
+        this.mockMvc.perform(get("/webjars")).andExpect(status().is4xxClientError());
```

Comment:
Why is only this one an error?

Reply:
because currently we are not using webjars but in the future we might